contrib/osx/packages.el

```diff
@@ -11,6 +12,39 @@
           dired-listing-switches "-aBhl --group-directories-first")
   (setq dired-use-ls-dired nil))
 
+(defun osx/init-launchctl ()
+  (use-package launchctl
+    :defer t
+    :init
+    (progn
+      (add-to-list 'auto-mode-alist '("\\.plist$" . nxml-mode))
+      (evil-leader/set-key "al" 'launchctl))
+    :config
+    (progn
+      ;; evilified mapping
+      (evilify launchctl-mode launchctl-mode-map
```

Comment:
Just a drive-by comment about the keybindings as some look a bit odd on the surface. Haven't tried any of them. Would the following make any more sense?

 `s` -> tabulated-list-sort
 `S` -> launchctl-start
 `k` -> launchctl-stop (think 'kill') -- or maybe K to match start
 `R` -> launchctl-restart
 `K` -> launchctl-remove (think 'KILL') -- or maybe D as in 'DELETE'
 `E` -> launchctl-enable
 `D` -> launchctl-disable
 `f` -> launchctl-filter
 `=` -> launchctl-setenv